tools/process-disclosure/contains-a-valid-directory.js

```diff
@@ -14,22 +14,28 @@ const vulnerabilityJson = await fs
     );
   });
 
+// Get required key values
 const packageId = vulnerabilityJson.PackageVulnerabilityID
 const packageName = vulnerabilityJson.Package.Name
 const packageRegistry = vulnerabilityJson.Package.Registry
```

Comment:
Perhaps a good opportunity to use [Object Destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)